tests/xapi-plugins/plugin_smartctl/test_smartctl.py

```diff
@@ -0,0 +1,19 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host > 8.2.
```

Comment:
Putthing the "Requirements" comment before the imports may be a good idea, but it's not consistent with the rest of the test scripts, so such a file layout change should not happen in this PR.

Reply:
done